edxval/utils.py

```diff
@@ -150,18 +150,37 @@ def video_image_path(video_image_instance, filename):  # pylint:disable=unused-a
     return '{}{}'.format(settings.VIDEO_IMAGE_SETTINGS.get('DIRECTORY_PREFIX', ''), filename)
 
 
+def get_configured_storage(settings_key):
+    """
+    Generic function to return a configured Django storage backend
+    based on the settings dictionary at `settings_key`.
+    """
+    config = getattr(settings, settings_key, {})
+    # Retrieve the storage class path and kwargs from the settings
+    storage_class_path = config.get('STORAGE_CLASS', {})
```

Comment:
The class should not return an empty dict, it should return none like we used to previously in `get_video_image_storage`